static/src/javascripts/projects/commercial/modules/prebid/bid-config.spec.js

```diff
@@ -674,3 +674,39 @@ describe('triplelift adapter', () => {
         });
     });
 });
+
+describe('xaxis adapter', () => {
+    beforeEach(() => {
+        resetConfig();
+        config.set('page.contentType', 'Article');
+        config.set('switches.prebidXaxis', true);
+        shouldIncludeXaxis.mockReturnValue(true);
+    });
+
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    test('should include xaxis adapter if condition is true ', () => {
```

Comment:
I suggest adding another test checking what happens when the condition is false to improve test coverage.